lib/emque/consuming/adapters/rabbit_mq/error_worker.rb

```diff
@@ -50,14 +50,13 @@ def retry_message(delivery_info, metadata, payload)
             begin
               logger.info "#{log_prefix} processing message #{metadata}"
               logger.debug "#{log_prefix} payload #{payload}"
-              message = Emque::Consuming::Message.new(
-                :original => payload
-              )
-              ::Emque::Consuming::Consumer.new.consume(:process, message)
+              message = Oj.load(payload)
+              topic = message.fetch(:metadata).fetch(:topic)
+              headers = metadata[:headers] || {}
+              x  = channel.default_exchange
```

Comment:
👋 Should we rename this `exchange`, rather than `x`?

Reply:
No.

Reply:
Wow, this is awesome, you guys made my day 😄 .

Reply:
plus, I made a change to eliminate that variable altogether